applications/serial_lte_modem/src/slm_at_fota.c

```diff
@@ -20,11 +20,12 @@
 
 LOG_MODULE_REGISTER(slm_fota, CONFIG_SLM_LOG_LEVEL);
 
-/* file_uri: scheme://hostname[:port]path */
-#define FILE_URI_MAX	256
-#define SCHEMA_HTTP	"http"
+/* file_uri: scheme://hostname[:port]path[?parameters] */
+#define FILE_URI_MAX	2048
+#define SCHEMA_HTTP		"http"
 #define SCHEMA_HTTPS	"https"
-#define URI_HOST_MAX	64
+#define URI_HOST_MAX	CONFIG_DOWNLOAD_CLIENT_MAX_HOSTNAME_SIZE
+#define URI_SCHEMA_MAX	9
```

Comment:
Why 9? Thought scheme to be "http" or "https" only

Reply:
That's true, I'll fix.